user/for-beginners.md

```diff
@@ -67,15 +67,15 @@ state that is not *passed*:
 Our [Common Builds Problems](/user/common-build-problems/) page is a good place
 to start troubleshooting why your build is broken.
 
-## Infrastructure and environment notes
+## Infrastructure and Environment Notes
 
 Travis CI offers a few different infrastructure environments, so you can select
 the setup that suits your project best:
 
 * *Ubuntu Linux* - these Linux Ubuntu environments run inside full virtual machines, provide plenty of computational resources, and support the use of `sudo`, `setuid`, and `setgid`.
-* *macOS* - uses one of several versions of the macOS operating system. This environment is useful for building projects that require the macOS software, such as projects written in Swift. It is not a requirement to use the macOS environment if you develop on a macOS machine.
+* *macOS* - uses one of several versions of the macOS operating system. This environment is useful for building projects that require the macOS software, such as projects written in Swift. It is not a requirement to use the macOS environment, if you develop on a macOS machine.
```

Comment:
I think it'd be helpful for us to link to the macOS reference page here, https://docs.travis-ci.com/user/reference/osx/, wdyt?